Makefile

```diff
@@ -23,3 +23,10 @@ lint:
 	-dotnet tool install -g dotnet-format
 	dotnet tool update -g dotnet-format
 	dotnet format
+
+.PHONY: restart-db
+restart-db:
```

Comment:
Nice! 🎉

Reply:
Could we add a description of what this helps to do in the readme? 
I've been bitten by old database docker images before and it took a bit of time to realise what the fix was at first.

Reply:
Yeah sure, I'll add it to base API at some point as well.